### DISCUSSION TITLE
TextTrimming="CharacterEllipsis"is not worked .

### DISCUSSION THREAD

**Discussion by LionelXN** [unoplatform/uno#15232] [Q&A]
![image](https://github.com/unoplatform/uno/assets/18021306/2d51808d-0cfc-4902-b642-9f91588d2403)
I set the property TextTrimming="CharacterEllipsis" to TextBlock, but it didn't go as expected, using '...' To replace the excess part

PS：My Uno version is 4.9.17, used in the GTK app

**Comment by jeromelaban** [comment]
Thanks for the report. Can you validate with the latest Uno? Thanks.
